gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Are we expecting to be running this script on other systems aside from the bootstrap VM?

If not, it might be cleaner to remove the cygwin/msys/darwin/nonstop stuff.

The only time that I'd think we could use this script is for running the style checks locally. However that can also be done by installing gradle locally.

Reply:
I recommend we don't touch this script at all; it comes from running `gradle wrapper`.

Reply:
I see, disregard this then.